tox.ini

```diff
@@ -1,10 +1,20 @@
 [tox]
-envlist = {py34, py35, py36, py37}
+envlist = {py34, py35, py36, py37}-nameko{2.6, 2.7, 2.8, 2.9, 2.10, 2.11}
 skipsdist = True
 
 [testenv]
 whitelist_externals = make
 usedevelop = true
 extras = dev
+deps =
+    nameko{2.6,2.7}: pytest<3.3.0
```

Comment:
I tried that, but it doesn't work. When adding a space here, `tox` interprets it as a dependency called `nameko{2.6, 2.7}: pytest<3.3.0` instead of as a directive indicating that it should only add `pytest<3.3.0` in test environments containing `nameko2.6` or `nameko2.7`.